builder/bin/publish-release-controller.go

```diff
@@ -77,15 +78,12 @@ func main() {
 
 	res, err := client.Do(req)
 
-	if res.StatusCode == 404 {
-		fmt.Println("Check the Controller. Is it running?")
-		os.Exit(1)
+	if err != nil {
+		log.Fatalln(err)
 	}
 
-	if err != nil || res.StatusCode != 200 {
-		fmt.Println("failed retrieving config from controller")
-		fmt.Println(res.Body)
-		os.Exit(1)
+	if res.StatusCode != 200 {
+		log.Fatalf("failed retrieving config from controller: %s\n", res.Body)
```

Comment:
Cool, that looks fine to me then. I still kind of like the explicit check for 404, as that's a clue that the controller is dead.